src/Extensions/ScheduledExecutionExtension.php

```diff
@@ -54,6 +54,11 @@ class ScheduledExecutionExtension extends DataExtension
      */
     public function updateCMSFields(FieldList $fields)
     {
+        $fields->removeByName('ExecuteInterval');
+        $fields->removeByName('ExecuteEvery');
+        $fields->removeByName('ExecuteFree');
+        $fields->removeByName('FirstExecution');
```

Comment:
Minor thing, but you can pass an array to `removeByName`

Reply:
Mind blown.